python/function.py

```diff
@@ -377,10 +380,18 @@ def __repr__(self):
 		return "<undetermined>"
 
 	def __eq__(self, other):
-		if self.type in [RegisterValueType.ConstantValue, RegisterValueType.ConstantValue] and isinstance(other, numbers.Integral):
```

Comment:
You should probably add this as its the most idiomatic way of implementing a __eq__ override. This also gives "other.__eq__" a chance to handle the same comparison.
```
		if not isinstance(other, self.__class__):
			return NotImplemented
```